src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -1291,6 +1295,7 @@ internal static new TextSpan GetDiagnosticSpan(SyntaxNode node, EditKind editKin
                 case SyntaxKind.BracketedParameterList:
                     if (editKind == EditKind.Delete)
                     {
+                        RoslynDebug.Assert(node.Parent is object);
                         return TryGetDiagnosticSpanImpl(node.Parent, editKind);
```

Comment:
>node.Parent [](start = 56, length = 11)

I think it'd be better to consistently use `Parent!` rather than adding asserts in some places and `!` in others.